[DOCS] Threat indicator rules and value list exceptions (#2058)
elastic/security-docs: docs/release-notes/8.2.asciidoc

[[known-issue-8.2.0]]
==== Known issues
* On macOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs), users might experience network connection issues. To resolve this issue, upgrade to macOS 12.4 or later.

> * Indicator match rules will encounter execution errors if they use an index that was generated by a value list ({pull}133457[#133457]).

Author: ```suggestion
* Avoid using indices populated from value lists for indicator match rules. Indicator match rules cannot use the `.lists` system index and will encounter execution errors when ran ({pull}133457[#133457]).
```

> * Avoid using indices populated from value lists for indicator match rules. Indicator match rules cannot use the `.lists` system index and will encounter execution errors when ran ({pull}133457[#133457]).

Author: ```suggestion
* Indicator match rules cannot use the `.lists` system index and will encounter execution errors when run. Avoid using indices populated from value lists for indicator match rules ({pull}133457[#133457]).
```

> * Indicator match rules cannot use the `.lists` system index and will encounter execution errors when run. Avoid using indices populated from value lists for indicator match rules ({pull}133457[#133457]).

Author: ```suggestion
* Indicator match rules cannot use the `.items` system index and will encounter execution errors when run. Avoid using indices populated from value lists for indicator match rules ({pull}133457[#133457]).
```

> * Indicator match rules cannot use the `.items` system index and will encounter execution errors when run. Avoid using indices populated from value lists for indicator match rules ({pull}133457[#133457]).

Author: ```suggestion
* Indicator match rules cannot use the `.items-*` system index and will encounter execution errors when run. Avoid using indices populated from value lists for indicator match rules ({pull}133457[#133457]).
```
